src/test/java/kg/jarkyn/cobspecserver/WorkerTest.java

```diff
@@ -0,0 +1,23 @@
+package kg.jarkyn.cobspecserver;
+
+import kg.jarkyn.cobspecserver.doubles.ClientDouble;
+import kg.jarkyn.cobspecserver.doubles.ResponseControllerDouble;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class WorkerTest {
```

Comment:
I have looked into when IOException can be thrown and it seemed the cases in which it may occur it is justifiable to have a RuntimeException. Ideally there could be something more elegant, a combination of logging for developer and an error message to the user, however this is a little beyond my stories.